pyproject.toml

```diff
@@ -39,13 +39,14 @@ pyqir = { version = "^0.8.0a1", allow-prereleases = true }
 lark = "^1.1.2"
 regex = "^2022.6.2"
 jsonpickle = "^2.2.0"
+pip-audit = "^2.7.3"
 
 [tool.poetry.group.dev.dependencies]
 coverage = "^6.3.2"
 pytest-cov = "~=2.10.1"
 autoflake = "^2.2.0"
 isort = "^5.13.2"
-black = "24.2.0"
+black = "24.3.0"
```

Comment:
If this affects the formatting you'll need to also adjust the version used in pre-commits.